static/styles/recent_topics.scss

```diff
@@ -60,11 +60,16 @@
         td {
             vertical-align: middle;
             padding: 3px 8px 3px 8px;
+
+            &:focus-within {
+                border-left: 2px solid hsl(215, 84%, 62%);
+            }
```

Comment:
I'm not sure this feels clear enough UI; maybe we want a border on all sides?  Might require a bit of a tweak for the "Actions" column too; ideally, the box would just be around the action.

Finally, the way these borders work ends up moving the topic text when you cursor into a row, which I think we don't want.